feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go

```diff
@@ -0,0 +1,324 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package setup is scoped only to be used for scripts in path
+// feature/experimental/system/gnmi/benchmarking/ate_tests/
+// Do not use elsewhere.
+package setup
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygnmi/ygnmi"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// Some of the variables defined below like DutAS, AteAS, PeerGrpName
+// RouteCount and IsisInstance are used in other files which import
+// setup.go
+const (
+	// DutAS can be exported
+	DutAS = 64500
+	// AteAs can be exported
+	AteAS  = 64501
+	ateAS2 = 64502
+	// PeerGrpName can be exported
+	PeerGrpName    = "BGP-PEER-GROUP"
+	plenIPv4       = 30
+	dutStartIPAddr = "192.0.2.1"
+	ateStartIPAddr = "192.0.2.2"
+	// RouteCount can be exported
+	RouteCount            = 200
+	advertiseBGPRoutesv4  = "203.0.113.1"
+	authPassword          = "ISISAuthPassword"
+	advertiseISISRoutesv4 = "198.18.0.0"
+	IsisInstance          = "DEFAULT"
```

Comment:
If IsisInstance is meant to exported, rename to ISISInstance; otherwise lowercase